components/measurement/nettests/WebConnectivity.js

```diff
@@ -244,6 +250,12 @@ const QueryContainer = ({query}) => {
                     ? dnsAnswer.hostname
                     : null // for any other answer_type, DATA column will be empty
               }
+              anc={dnsAnswer.asn
+                   ? dnsAnswer.as_org_name
+                     ? `AS${dnsAnswer.asn} (${dnsAnswer.as_org_name})`
+                     : `AS${dnsAnswer.asn}`
+                   : dnsAnswer.as_org_name
+                   ? `AS??? (${dnsAnswer.as_org_name})` : null}
```

Comment:
Thanks for the suggestion.  I have now moved this code to a separate function, `dnsAnswerIpInfo(dnsAnswer)`.  This could be extended to flag private addresses as suggested below.